src/web/utils/utils.ts

```diff
@@ -0,0 +1,208 @@
+    props.onPressOut ||
+    props.onLongPress
+  );
+}
+
+export function hasResponderEvents(props: BaseProps): boolean {
+  return !!(
+    props.onResponderGrant ||
+    props.onResponderMove ||
+    props.onResponderRelease ||
+    props.onResponderTerminate ||
+    props.onResponderTerminationRequest ||
+    props.onMoveShouldSetResponder ||
+    props.onMoveShouldSetResponderCapture ||
+    props.onResponderEnd ||
+    props.onResponderReject ||
+    props.onResponderStart ||
+    props.onScrollShouldSetResponder ||
+    props.onScrollShouldSetResponderCapture ||
+    props.onSelectionChangeShouldSetResponder ||
+    props.onSelectionChangeShouldSetResponderCapture ||
+    props.onStartShouldSetResponder ||
+    props.onStartShouldSetResponderCapture
+  );
+}
+
+export function stringifyTransformProps(transformProps: TransformProps) {
+  const transformArray = [];
+  if (transformProps.translate != null) {
+    transformArray.push(`translate(${transformProps.translate})`);
+  }
+  if (transformProps.translateX != null || transformProps.translateY != null) {
+    transformArray.push(
+      `translate(${transformProps.translateX || 0}, ${
+        transformProps.translateY || 0
+      })`
+    );
+  }
+  if (transformProps.scale != null) {
+    transformArray.push(`scale(${transformProps.scale})`);
+  }
+  if (transformProps.scaleX != null || transformProps.scaleY != null) {
+    transformArray.push(
+      `scale(${transformProps.scaleX || 1}, ${transformProps.scaleY || 1})`
+    );
+  }
+  // rotation maps to rotate, not to collide with the text rotate attribute (which acts per glyph rather than block)
+  if (transformProps.rotation != null) {
+    transformArray.push(`rotate(${transformProps.rotation})`);
+  }
+  if (transformProps.skewX != null) {
+    transformArray.push(`skewX(${transformProps.skewX})`);
+  }
+  if (transformProps.skewY != null) {
+    transformArray.push(`skewY(${transformProps.skewY})`);
+  }
+  return transformArray;
+}
+
+export const camelCaseToDashed = (camelCase: string) => {
+  return camelCase.replace(/[A-Z]/g, (m) => '-' + m.toLowerCase());
+};
+
+export const getBoundingClientRect = (node: SVGElement) => {
+  if (node) {
+    const isElement = node.nodeType === 1; /* Node.ELEMENT_NODE */
+    if (isElement && typeof node.getBoundingClientRect === 'function') {
+      return node.getBoundingClientRect();
+    }
+  }
+  throw new Error('Can not get boundingClientRect of ' + node || 'undefined');
+};
+
+export const remeasure = (element: HTMLElement | null) => {
+  if (!element) return;
+  const rect = element.getBoundingClientRect();
+  return {
+    width: rect.width,
+    height: rect.height,
+    top: rect.top + window.scrollY,
+    left: rect.left + window.scrollX,
+  };
+};
+
+/**
+ * `react-native-svg` supports additional props that aren't defined in the spec.
+ * This function replaces them in a spec conforming manner.
+ */
+interface PrepareComponentProps extends CreateComponentProps {
```

Comment:
```suggestion
interface PreparedComponentProps extends CreateComponentProps {
```

I think that verbs are not the best names for interfaces 😅

Reply:
@m-bert maybe is better `AdvancedComponentProps` ? :D

Reply:
I think `prepared` fits better, but it's up to you 😅